commitlint.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	parserPreset: 'conventional-changelog-conventionalcommits',
+	rules: {
+		'body-leading-blank': [1, 'always'],
+		'footer-leading-blank': [1, 'always'],
+		'header-max-length': [2, 'always', 72],
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2 tabs. (indent)](https://app.codacy.com/app/PeterNgTr/codeceptjs-bshelper/pullRequest?prid=4722635)